packages/ui/src/views/repo/repo-tags/repo-tags-list-page.tsx

```diff
@@ -104,7 +104,7 @@ export const RepoTagsListView: FC<RepoTagsListViewProps> = ({
         <Spacer size={2} />
         {(isLoading || !!tagsList.length || isDirtyList) && (
           <>
-            <span className="text-24 font-medium text-cn-foreground-1">{t('views:repos.tags', 'Tags')}</span>
+            <span className="text-24 text-cn-foreground-1 font-medium">{t('views:repos.tags', 'Tags')}</span>
```

Comment:
This looks like some kind of heading, but a `span` was used. Can we change this to something more appropriate?

Reply:
Replaced with `<h1 />`